src/Change.php

```diff
@@ -33,7 +29,6 @@ final class Change
 
     private function __construct(string $modificationType, string $description, bool $isBcBreak)
     {
-        Assert::that($modificationType)->inArray(self::$validModificationTypes);
```

Comment:
Why removing this validation? Doesn't seem to be any alternative type safety being added instead..?

Reply:
All of the API is `private`, so this assertion was not adding any further safety: no public API allows for invalid states

Reply:
OK gotcha